scripts/close_boundary.py

```diff
@@ -5,41 +5,62 @@
 import matplotlib.pyplot as plt
 import argparse
 
+
 def clip_to_value(arr, test_arr=None, value=0.0):
     if test_arr is None:
         test_arr = arr
 
-    arr[test_arr>value] = value
+    arr[test_arr > value] = value
 
 
 def main():
     # Handle arguments
-    parser = argparse.ArgumentParser(description='Add halo surrounding given netcdf file')
-    parser.add_argument('--input', required=True, help='input file')
-    parser.add_argument('--output', required=True, help='output file')
-    parser.add_argument('--value', type=float, default=0.0, help='value to replace boundary with')
-    parser.add_argument('--offset', action='store_true', default=False, help='use one index from true boundary as boundary')
-    parser.add_argument('--plot', action='store_true', default=False, help='plot final grid before saving')
-    parser.add_argument('--region', required=False, metavar=('xmin', 'xmax', 'ymin', 'ymax'), nargs=4, type=float,
-                        help='output region. Defaults to the extent of the input grid.')
+    parser = argparse.ArgumentParser(
+        description="Add halo surrounding given netcdf file"
```

Comment:
```suggestion
        description="Add piecewise halo surrounding given netcdf file"
```
Just a suggestion, not sure it's the best. I wonder whether just "halo" implies continuity all around the grid.